index.js

```diff
@@ -80,20 +80,29 @@ prog
           acorn.parse(code, acornOpts)
         } catch (err) {
           logger.debug(`ES-Check: failed to parse file: ${file} \n - error: ${err}`)
-          errFiles.push(file)
+          const errorObj = {
```

Comment:
why not just do:

```js
err.file = file
errArray.push(err)
```

Reply:
![1__jwainwright_dsc-ml-jwainwright____dropbox_me_git-docs_oss_es-check__zsh_](https://user-images.githubusercontent.com/1074042/35705558-a4ab4194-0757-11e8-9d0a-36b4098e0751.jpg)

Reply:
^ @jonathanong if you feel no pretty formatting is better, I'm cool wit' dat!

Reply:
`err.stack.replace(/\n/g, '\n  ')` or whatever indentation you want